docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -63,8 +68,8 @@ You can now use the Helm chart to deploy the ACK service controller for Amazon R
 For example, to specify that the RDS API calls go to the `us-east-1` region, you can deploy the service controller with the following command:
 
 ```bash
-helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/rds-chart \
-  --version=v0.0.27 --generate-name --set=aws.region=us-east-1
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
```

Comment:
```suggestion
aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
```
```suggestion
aws ecr-public get-login-password --region "${AWS_REGION}" | helm registry login --username AWS --password-stdin public.ecr.aws
```

Reply:
When authenticating to a public registry, always authenticate to the us-east-1 Region when using the AWS CLI. https://docs.aws.amazon.com/AmazonECR/latest/public/public-registries.html